lib/chef/provider/systemd_unit.rb

```diff
@@ -236,7 +236,8 @@ def enabled?
       def masked?
         # Note: masked-runtime is excluded, because runtime is volatile, and
         # because masked-runtime is not masked.
-        systemd_unit_status["UnitFileState"] == "masked"
+        # Unit status of a masked resource is "bad"
```

Comment:
are you saying "masked" was never valid and should just be bad? I don't think that's true. I think what you're saying is that "bad" is similar to masked?